build_variables.bzl

```diff
@@ -1154,6 +1154,7 @@ aten_native_source_codegen_list = [
     "aten/src/ATen/native/cpu/layer_norm_kernel.cpp",
     "aten/src/ATen/native/cpu/scaled_modified_bessel_k1.cpp",
     "aten/src/ATen/native/cpu/spherical_bessel_j0.cpp",
+    "aten/src/ATen/native/cpu/SparseFactories.cpp",
```

Comment:
Only now spotted that: shouldn't this file be in a single list? To codegen, that is.